analyzers/tests/ITs.JsonParser.Test/JsonParsingTest.cs

```diff
@@ -1,12 +1,187 @@
-﻿namespace SonarAnalyzer.Net8.Test;
+﻿namespace ITs.JsonParser.Test;
 
 [TestClass]
 public class JsonParsingTest
 {
-    //TODO: Add tests during implementation
+    public TestContext TestContext { get; set; }
+
     [TestMethod]
-    public void NewIssueReports_Test()
+    public void IssueParser_Execute_Single()
+    {
+        var root = TestDirectory();
+        var inputPath = Path.Combine(root, "read");
+        var outputPath = Path.Combine(root, "write");
+        Directory.CreateDirectory(Path.Combine(inputPath, "solution"));
+        Expected.CreateSarifReport(inputPath, "solution", "project", "net6.0", Expected.CreateIssue("S100", "Message_1", "foo/bar/File1.cs", 1, 1));
+        var outFile = Path.Combine(outputPath, "solution", "S100-project-net6.0.json");
+
+        IssueParser.Execute(inputPath, outputPath);
+
+        VerifyResultFile(outFile, """
+            {
+              "Issues": [
+                {
+                  "Id": "S100",
+                  "Message": "Message_1.",
+                  "Location": {
+                    "Uri": "https://github.com/SonarSource/sonar-dotnet/blob/master/analyzers/its/foo/bar/File1.cs#L1",
+                    "Region": {
+                      "StartLine": 1,
+                      "StartColumn": 42,
+                      "EndLine": 1,
+                      "EndColumn": 69
+                    }
+                  }
+                }
+              ]
+            }
+            """);
+    }
+
+    [TestMethod]
+    public void IssueParser_Execute_Multiple()
```

Comment:
This is more like an integration test, I don't want to split it because I want to see that the ETL pipeline works correctly and everything ends up where it's supposed to be.